sdk/schemaregistry/schema-registry-json/src/cache.ts

```diff
@@ -66,3 +54,10 @@ export function getSchemaObject(schema: string): SchemaObject {
     `Parsing Json schema failed:\n\n\t${schema}\n\nSee 'cause' for more details.`
   );
 }
+
+export interface SchemaObject {
+  id?: string;
+  $id?: string;
+  $schema?: string;
+  [x: string]: any;
```

Comment:
do we need the indexer? and if so, should the value be `unknown` instead of `any`?

Reply:
I use the indexer since it specifies that there can be more properties. I'll change to `unknown` for safer type checking.

Reply:
if we don't need to access any additional properties, we don't actually have to type them. It doesn't appear this is a public type either?

Reply:
That makes sense. I will make the update accordingly.

Reply:
That makes sense, I'll remove that.